.github/workflows/traffic-ops.yml

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-name: Traffic Ops Go client/API integration tests
+name: TO Integration Tests
```

Comment:
The cross-app integration being tested here is TO's API with its Go client. TO is basically just an API, so saying API may be redundant, and it can be abbreviated from Traffic Ops to TO, but imo this should still mention the Go client.

Reply:
Is Go necessary? Do we have another client?

Reply:
Yes, we have clients for the TO API in two other languages: Java and Python.